gradle/versions.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     version = '0.1.0'
 
     gradle_plugin ='3.2.1'
-    kotlin_version = '1.2.71'
+    kotlin_version = '1.3.0'
```

Comment:
Lets change it separately after the 0.1.1 release?
The same with mapbox dependencies.